module/str_format_conv.go

```diff
@@ -0,0 +1,15 @@
+package module
```

Comment:
I don't see a need for this. The pgs packages provide helpers for different cases.